src/components/Atoms/Controls/Input/Input.stories.tsx

```
@@ -49,7 +49,7 @@ export const Email: Story = {
     label: "Email Input",
     placeholder: "Enter email",
   },
-  render: (args) => <InputWrapper {...args} />,
+  render: (args: any) => <InputWrapper {...args} />,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)